src/code-analysis/comment-collector.js

```diff
@@ -153,3 +156,43 @@ function applyMemberRoles(members, data) {
 
   return result;
 }
+
+/**
+ * @param {Area[]} members
+ */
+function collectExceptions(members) {
+  const result = [];
+
+  for (const member of members) {
+    const _member = { ...member };
```

Comment:
Is there a reason to do this instead of just pushing the "members" that make it thru to the bottom into the result array and returning it? 

Also can we name this something different because the tags are not guaranteed to be members at all?